<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chats</class>
 <widget class="QWidget" name="chats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="chat_bg" native="true">
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>1</y>
        <width>782</width>
        <height>519</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: rgb(44, 60, 75);
border-radius: 10px;
</string>
      </property>
      <widget class="QWidget" name="user_bg_3" native="true">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>20</y>
         <width>716</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(27, 31, 48);
border-radius: 20px;
</string>
       </property>
       <widget class="QGraphicsView" name="user_img_2">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGraphicsView {
    background-color: rgb(255, 255, 255);
    width: 25px; /* Set width and height to same value */
    height: 25px; /* Set width and height to same value */
    border-radius: 15px; /* Half of width/height for a circle */
}
</string>
        </property>
       </widget>
       <widget class="QLabel" name="user_name_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>17</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>null</string>
        </property>
       </widget>
       <widget class="QPushButton" name="callButton_2">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>13</y>
          <width>40</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="wardenfort.qrc">
          <normaloff>:/wardenfort/phone-call.png</normaloff>:/wardenfort/phone-call.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="findButton_2">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>12</y>
          <width>30</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="wardenfort.qrc">
          <normaloff>:/wardenfort/find.png</normaloff>:/wardenfort/find.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="optionButton_2">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>12</y>
          <width>30</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="wardenfort.qrc">
          <normaloff>:/wardenfort/dots.png</normaloff>:/wardenfort/dots.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="typeField_2">
       <property name="geometry">
        <rect>
         <x>43</x>
         <y>449</y>
         <width>671</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(27, 31, 48);
color: rgb(255, 255, 255);
border-radius: 20px;
padding-left: 20px;
padding-right:150px;</string>
       </property>
       <property name="placeholderText">
        <string>Type your message</string>
       </property>
      </widget>
      <widget class="QPushButton" name="uploadimgButton_2">
       <property name="geometry">
        <rect>
         <x>576</x>
         <y>464</y>
         <width>40</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wardenfort.qrc">
         <normaloff>:/wardenfort/photo.png</normaloff>:/wardenfort/photo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="emojiButton_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>463</y>
         <width>40</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wardenfort.qrc">
         <normaloff>:/wardenfort/happy.png</normaloff>:/wardenfort/happy.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="sendButton_2">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>465</y>
         <width>40</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wardenfort.qrc">
         <normaloff>:/wardenfort/send.png</normaloff>:/wardenfort/send.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="uploadfileButton_2">
       <property name="geometry">
        <rect>
         <x>659</x>
         <y>462</y>
         <width>40</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wardenfort.qrc">
         <normaloff>:/wardenfort/upload (1).png</normaloff>:/wardenfort/upload (1).png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QListWidget" name="chatHistory">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>80</y>
         <width>716</width>
         <height>361</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>150</y>
         <width>151</width>
         <height>211</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
     <widget class="QLabel" name="searching">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>22</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="wardenfort.qrc">:/wardenfort/search.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchField">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>17</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;    
background-color: rgb(44, 60, 75);
border-radius: 10px;
padding-left: 10px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>search</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>256</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #191F32;
border-radius: 20px;
</string>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
     </widget>
     <widget class="QWidget" name="user_bg" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>519</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #191F32;
border-radius: 20px;
</string>
      </property>
     </widget>
     <zorder>chat_bg</zorder>
     <zorder>user_bg</zorder>
     <zorder>searchField</zorder>
     <zorder>searching</zorder>
     <zorder>listWidget</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="wardenfort.qrc"/>
 </resources>
 <connections/>
</ui>
